### PR TITLE
Adding Style section on readme

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
As title says

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,41 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
 Also, you can configure the product summary that is defined on minicart. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API. 
 
 ### Styles API
-:construction: :construction: :construction:
+This apps has CSS customization through `CSS Modules`.
```
Comment (GustavoMA) [inline README.md]: `This app` :hammer:

File: README.md [hunk 2]
```diff
@@ -67,7 +67,41 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
 Also, you can configure the product summary that is defined on minicart. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API. 
 
 ### Styles API
-:construction: :construction: :construction:
+This apps has CSS customization through `CSS Modules`. 
+
+This apps has CSS customization through `CSS Modules`. CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules [here](https://github.com/css-modules/css-modules) .
```
Comment (GustavoMA) [inline README.md]: `This app` :hammer:

File: README.md [hunk 3]
```diff
@@ -67,7 +67,41 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
...
+Below, we describe the tokens, their explanation and the component where it is located.
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | `index`            | The main container of minicart                         |
+| `label`            | `index`            | Minicart icon label                                    |
+| `badge`            | `index`            | Minicart badge with the product quantity on it         |
+| `content`          | `MinicartContent`  | Minicart products container                            |
+| `contentSmall`     | `MinicartContent`  | Minicart content for small sidebar                     |
+| `contentLarge`     | `MinicartContent`  | Minicart content for small sidebar                     |
```
Comment (GustavoMA) [inline README.md]: Fix description.

File: README.md (hunk 1 above)
Comment (brunojdo) [inline README.md]: duplicated

File: README.md [hunk 4]
```diff
@@ -67,7 +67,41 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
...
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and version. Like `container` token declared in minicart, generate the classname `vtex.minicart-2-x-container`.
+
+Below, we describe the tokens, their explanation and the component where it is located.
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | `index`            | The main container of minicart                         |
+| `label`            | `index`            | Minicart icon label                                    |
+| `badge`            | `index`            | Minicart badge with the product quantity on it         |
+| `content`          | `MinicartContent`  | Minicart products container                            |
```
Comment (GustavoMA) [inline README.md]: Shouldn be `Minicart products'` or `Minicart product's` ?

**Review by GustavoMA — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (brunojdo) [inline README.md]: app

File: README.md [hunk 5]
```diff
@@ -67,7 +67,41 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
 Also, you can configure the product summary that is defined on minicart. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API. 
 
 ### Styles API
-:construction: :construction: :construction:
+This apps has CSS customization through `CSS Modules`. 
+
+This apps has CSS customization through `CSS Modules`. CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules [here](https://github.com/css-modules/css-modules) .
+
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and version. Like `container` token declared in minicart, generate the classname `vtex.minicart-2-x-container`.
```
Comment (brunojdo) [inline README.md]: and major version

File: README.md [hunk 6]
```diff
@@ -67,7 +67,40 @@ Through the Storefront, you can change the minicart's behavior and interface. Ho
...
+
+CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules [here](https://github.com/css-modules/css-modules) .
+
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and version. Like `container` token declared in minicart, generate the classname `vtex.minicart-2-x-container`.
+
+Below, we describe the tokens, their explanation and the component where it is located.
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | `index`            | The main container of minicart                         |
```
Comment (brunojdo) [inline README.md]: add the component link [index](github link)

**Review by brunojdo — CHANGES_REQUESTED** [review]

**Review by GustavoMA — APPROVED** [review]

**Review by brunojdo — APPROVED** [review]

**Pull request merged**